book/2.json

```diff
@@ -1,42 +1,42 @@
 [
     {"text":""},
     {"text":"     More Options","color":"dark_gray"},
-    {"text":"\nMarkers ","color":"dark_gray"},
+    {"text":"\nMarkers ","color":"dark_gray","hoverEvent":{"action":"show_text","value":"Sends a message in chat\nwhenever the set amount of\ntime has passed"}},
     {"storage":"uhc_pack:text","nbt":"Icon.Marker","interpret":true,"hoverEvent":{"action":"show_text","value":"Click to toggle between enabled and disabled"},"clickEvent":{"action":"run_command","value":"/trigger uhcOpt set 108"}},
     {"text":" \u2503 "},
     {"text":"-","color":"red","hoverEvent":{"action":"show_text","value":"-10 minutes"},"clickEvent":{"action":"run_command","value":"/trigger uhcOpt set 100"}},
     {"text":" "},
     {"score":{"name":"UHC","objective":"uhcMM"},"color":"gray"},
     {"text":"min ","color":"gray"},
     {"text":"+","color":"dark_green","hoverEvent":{"action":"show_text","value":"+10 minutes"},"clickEvent":{"action":"run_command","value":"/trigger uhcOpt set 101"}},
-    {"text":"\nEternal ","color":"dark_gray"},
+    {"text":"\nEternal ","color":"dark_gray","hoverEvent":{"action":"show_text","value":"Sets the time to day\nand stops the daylight cycle\nafter the set time"}},
     {"storage":"uhc_pack:text","nbt":"Icon.Eternal","interpret":true,"hoverEvent":{"action":"show_text","value":"Click to toggle between enabled and disabled"},"clickEvent":{"action":"run_command","value":"/trigger uhcOpt set 106"}},
     {"text":" \u2503 "},
     {"text":"-","color":"red","hoverEvent":{"action":"show_text","value":"-10 minutes"},"clickEvent":{"action":"run_command","value":"/trigger uhcOpt set 102"}},
     {"text":" "},
     {"score":{"name":"UHC","objective":"uhcEter"},"color":"gray"},
     {"text":"min ","color":"gray"},
     {"text":"+","color":"dark_green","hoverEvent":{"action":"show_text","value":"+10 minutes"},"clickEvent":{"action":"run_command","value":"/trigger uhcOpt set 103"}},
-    {"text":"\nGlowing ","color":"dark_gray"},
+    {"text":"\nGlowing ","color":"dark_gray","hoverEvent":{"action":"show_text","value":"Makes any alive players glow,\nmaking them visible to the\nother teams"}},
```

Comment:
Change 'any alive players' to 'all players'. I think the technical difference is not relevant and it makes it read a little better.